examples/walkthrough/README.md

```diff
@@ -1,73 +1,149 @@
 # Kubernetes 101 - Walkthrough
 
+## Kubectl CLI
+
+The easiest way to interact with Kubernetes is via the command-line interface.
+
+If you downloaded a pre-compiled release, kubectl should be under `platforms/<os>/<arch>`.
+
+If you built from source, kubectl should be either under `_output/local/bin/<os>/<arch>` or `_output/dockerized/bin/<os>/<arch>`.
+
+### Install
```

Comment:
Seems like a style complaint about how the github header levels don't look different enough. I'm not sure skipping a header level really solves the problem...